### PR TITLE
Update FOSUserBundle.fr.yml

### PR DESCRIPTION
**Pull request by Laurent3170**
Change message for ressetting password according to en version

### CODE REVIEW

**Comment by Laurent3170** [comment]
Yes I saw it was like this in the english version but "\n" are used in the french one. Should I update the whole french version according to the english one ?

File: Resources/translations/FOSUserBundle.fr.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ registration:
         message: "Bonjour %username% !\n\nPour valider votre compte utilisateur, merci de vous rendre sur %confirmationUrl%\n\nCordialement,\nL'équipe.\n"
 resetting:
     password_already_requested: 'Un nouveau mot de passe a déjà été demandé pour cet utilisateur dans les dernières 24 heures.'
-    check_email: 'Un e-mail a été envoyé à l''adresse %email%. Il contient un lien sur lequel il vous faudra cliquer afin de réinitialiser votre mot de passe.'
+    check_email: 'Un e-mail a été envoyé. Il contient un lien sur lequel il vous faudra cliquer pour réinitialiser votre mot de passe.\nRemarque: Vous ne pouvez demander un nouveau mot de passe que toutes les %tokenLifetime% heures.\n\nSi vous ne recevez pas un email, vérifiez votre dossier spam ou essayez à nouveau.'
```
Comment (Laurent3170) [inline Resources/translations/FOSUserBundle.fr.yml]: Yes I saw it was like this in the english version but "\n" are used in the french one. Should I update the whole french version according to the english one ?

Comment (XWB) [inline Resources/translations/FOSUserBundle.fr.yml]: Yes please :)

Comment (Laurent3170) [inline Resources/translations/FOSUserBundle.fr.yml]: Done. I have rewieved the whole file and put single quote (') everywhere even if english version sometimes have and sometimes not

Comment (XWB) [inline Resources/translations/FOSUserBundle.fr.yml]: :+1:

**Pull request merged**